android/app/src/main/java/com/pccontrol/voice/network/PCDiscovery.kt

```
@@ -247,35 +247,141 @@ class PCDiscovery(private val context: Context) {
+                         discoveredPCs.add(
+                            DiscoveredPC(
+                                id = "pc_${ip.replace(".", "_")}",
+                                name = "PC Agent (SSDP)",
+                                ipAddress = ip,
+                                port = PC_AGENT_PORT,
+                                isOnline = true,
+                                lastSeen = System.currentTimeMillis()
+                            )
+                         )
+                    }
+                } catch (e: SocketTimeoutException) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)